Packs/IdentityRecordedFuture/Integrations/IdentityRecordedFuture/IdentityRecordedFuture.py

```diff
@@ -87,63 +137,150 @@ def __init__(self, rf_client: Client):
 
     def identity_search_command(self):
-        response = self.client.identity_search()
+        response = self.client.credentials_search()
         return self._process_result_actions(response=response)
 
     def identity_lookup_command(self):
         """Lookup command for identities"""
-        response = self.client.identity_lookup()
+        response = self.client.credentials_lookup()
         return self._process_result_actions(response=response)
 
     def password_lookup_command(self):
         """Lookup command for passwords"""
         response = self.client.password_lookup()
         return self._process_result_actions(response=response)
 
+    #######################################################
+    ################## Playbook alerts ####################
+    #######################################################
+
+    def fetch_incidents(self) -> None:
+
+        response = self.client.fetch_incidents()
+
+        if isinstance(response, CommandResults):
+            # 404 case.
+            return
+
+        for _key, _val in response.items():
+            if _key == "demisto_last_run":
+                demisto.setLastRun(_val)
+            if _key == "incidents":
+                _transform_incidents_attachments(_val)
+                demisto.incidents(_val)
+
+    def playbook_alert_search_command(self) -> Optional[List[CommandResults]]:
+        response = self.client.search_playbook_alerts()
+        return self._process_result_actions(response=response)
+
+    def playbook_alert_details_command(self) -> Optional[List[CommandResults]]:
+        response = self.client.details_playbook_alerts()
+        return self._process_result_actions(response=response)
+
+    def playbook_alert_update_command(self) -> Optional[List[CommandResults]]:
+        response = self.client.update_playbook_alerts()
+        return self._process_result_actions(response=response)
+
+
+def _transform_incidents_attachments(incidents: list) -> None:
+    for incident in incidents:
+        attachments = []
+        incident_json = json.loads(incident.get("rawJSON", "{}"))
+        if incident_json.get("panel_evidence_summary", {}).get("screenshots"):
+            for screenshot_data in incident_json["panel_evidence_summary"][
+                "screenshots"
+            ]:
+                file_name = (
+                    f'{screenshot_data.get("image_id", "").replace("img:", "")}.png'
+                )
+                file_data = screenshot_data.get("base64", "")
+                file = fileResult(file_name, base64.b64decode(file_data))
+                attachment = {
+                    "description": screenshot_data.get("description"),
+                    "name": file.get("File"),
+                    "path": file.get("FileID"),
+                    "showMediaFile": True,
+                }
+                attachments.append(attachment)
+            incident["attachment"] = attachments
+
+
+def get_client() -> Client:
```

Comment:
Why did you move the logic here from the main function?

Reply:
to make main() cleaner and easier to read

Reply:
Our best practice is to leave main as is.
All the param logic and the client is there.
But it is a suggestion